crates/snp_validation/src/config.rs

```diff
@@ -0,0 +1,179 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
+use crate::{PlatformInfoFlags, PolicyFlags};
+use flagset::FlagSet;
+use semver::Version;
+use serde::de::Error;
+use serde::{Deserialize, Deserializer, Serialize};
+use std::str::FromStr;
+
+#[derive(Clone, Deserialize, Debug, Default, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct Config {
+    /// Values for `author_key_digest` in the report body.
+    /// This is the list of public keys which have signed the signature of the Enarx binary.
+    #[serde(default)]
+    #[serde(deserialize_with = "from_hex")]
+    pub enarx_signer: Option<Vec<Vec<u8>>>,
+
+    /// Values for `id_key_digest` in the report body.
+    /// This is the list of public keys which have signed the Enarx binary.
+    #[serde(default)]
+    #[serde(deserialize_with = "from_hex")]
+    pub id_key: Option<Vec<Vec<u8>>>,
```

Comment:
Why not  `id_key_digest`? Do you need the real public key here?

Reply:
If both `enarx_signer` and `id_key` are empty, something might be wrong and SNP should not be attested.

Reply:
Maybe at a minimum, Steward should check the fields are populated (not zero?), and the config optionally specifies which.

Reply:
If any fields are empty, they are ignored.